pkg/security/admission/scc_exec_test.go

```diff
@@ -110,10 +110,5 @@ func TestExecAdmit(t *testing.T) {
 			t.Errorf("%s: no actions found", k)
 		}
 
-		if v.shouldHaveClientAction {
```

Comment:
is there a test making sure exec works if the SCC had rights and the user didn't?